airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -11,6 +11,12 @@
 from prometheus_client import generate_latest, REGISTRY
 from prometheus_client.core import GaugeMetricFamily
 from sqlalchemy import and_, func
+from airflow.configuration import conf
+from airflow_prometheus_exporter import xcom_config
+
+import pickle
+import json
+from airflow.utils.log.logging_mixin import LoggingMixin
```

Comment:
we try to follow the pep8 import rules to maintain the code as more readable (https://www.python.org/dev/peps/pep-0008/#imports) as well as maintaining alphabetical ordering

Reply:
ok